src/mscorlib/shared/System/Diagnostics/StackFrame.cs

```diff
@@ -0,0 +1,272 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System.Text;
+using System;
+using System.IO;
+using System.Reflection;
+using System.Diagnostics.Contracts;
```

Comment:
Removed [ProjectN] together with the clause

            Contract.Ensures(Contract.Result<MethodBase>() != null);

in GetMethod.